packages/expo/package.json

```diff
@@ -44,24 +57,26 @@
     "@clerk/shared": "2.3.2",
     "@clerk/types": "4.7.0",
     "base-64": "^1.0.0",
-    "react-native-url-polyfill": "2.0.0",
+    "react-native-url-polyfill": "^2.0.0",
     "tslib": "2.4.1"
   },
   "devDependencies": {
     "@clerk/eslint-config-custom": "*",
     "@types/base-64": "^1.0.2",
     "@types/node": "^20.11.24",
-    "@types/react": "*",
-    "@types/react-dom": "*",
+    "@types/react": "^18.2.0",
+    "@types/react-dom": "^18.2.0",
     "expo-auth-session": "^5.4.0",
     "expo-web-browser": "^12.8.2",
+    "react-native": "^0.74.2",
     "typescript": "*"
   },
   "peerDependencies": {
     "expo-auth-session": ">=4",
     "expo-web-browser": ">=12.5.0",
     "react": ">=18",
-    "react-dom": ">=18"
+    "react-dom": ">=18",
+    "react-native": ">=0.73"
```

Comment:
Can this new requirement break existing user apps that already use `@clerk/clerk-expo` and e.g. also have no intention of using Expo Web?

Reply:
Those have been added as we now need the `react-native` dep and the `react` one are added to override the momorepo's react version which does not match the requirement of the `react-native` dep. We had a problem with this as Renovate updates the momorepos React version but does not take account the Version needed for React Native

Reply:
Yeah, I get that 👍 
What I tried to ask is:

Assume we have a user that currently has the latest `@clerk/clerk-expo` installed. Their app uses `react-native@0.72`. They now install this new version (from this PR) and suddenly their app breaks (either during installation because of npm peerDep mismatch or during runtime because `Platform` is not accessible).

Reply:
Hmm you are right, let me check if we can lower that to `react-native@0.72` in order to avoid that, I changed it at first as it's a minor version change and I think that  nothing will break, but you are right that it might be caught on peerDep mismatches